### ISSUE TITLE
Form controls inside a ControlRow in a Dialog created using the ALF_CREATE_DIALOG_REQUEST not publishing to _valueChangeOf_ topic when form is rendered

### ISSUE THREAD

**Issue by johhnywishwash** [Alfresco/Aikau#999]
Form controls, that are inside a **_ControlRow_**, that are in a **_Form_**, that have been created in a Dialog using **ALF_CREATE_DIALOG_REQUEST** will not initially publish their value to the _valueChangeOf_ topic unless the Forms config has the the property **_"value:{}"_**.

I came across this as we are creating a dialog that contains a **_DynamicForm_** that is updated by a select control in the same dialog window.  I noticed that only the controls not inside **_ControlRows_** were publishing to the **_valueChangeOf_** topic.

A **_Form_** without the **_"value:{}"_** set but just on the page seems to work fine.

As a work around I have just included **_"value:{}"_** in the **_Form_** config
Here is an example demonstrating the behaviour.  If you filter the PubSub log by _valueChangeOf_ you will see Control C does not publish its value:

```
{
		name : "alfresco/forms/Form",
		config : {
			widgets : [ {
				name : "alfresco/forms/ControlRow",
				config : {
					title : "Control Row 1",
					widgets : [ {
						name : "alfresco/forms/controls/TextBox",
						config : {
							fieldId : "control_a",
							name : "control_a",
							label : "Control A"
						}
					} ]
				}
			}, {
				name : "alfresco/forms/controls/TextBox",
				config : {
					fieldId : "control_b",
					name : "control_b",
					label : "Control B"
				}
			} ]
		}
	}, {
		name : "alfresco/buttons/AlfButton",
		config : {
			label : "Show Form",
			publishTopic : "ALF_CREATE_DIALOG_REQUEST",
			publishPayload : {
				dialogId : "DIALOG",
				dialogTitle : "DIALOG FORM WITH CONTROL ROW",
				widgetsContent : [ {
					name : "alfresco/forms/Form",
					config : {
						//value : {},
						widgets : [ {
							name : "alfresco/forms/ControlRow",
							config : {
								title : "Control Row 2",
								widgets : [ {
									name : "alfresco/forms/controls/TextBox",
									config : {
										fieldId : "control_c",
										name : "control_c",
										label : "Control C"
									}
								} ]
							}
						}, {
							name : "alfresco/forms/controls/TextBox",
							config : {
								fieldId : "control_d",
								name : "control_d",
								label : "Control D"
							}
						} ]
					}
				} ],
				widgetsButtons : [ {
					name : "alfresco/buttons/AlfButton",
					config : {
						label : "Close dialog",
						publishTopic : "CUSTOM_TOPIC"
					}
				} ]
			}
		}
	}, {
		name : "alfresco/buttons/AlfButton",
		config : {
			label : "Show Form with value : {}",
			publishTopic : "ALF_CREATE_DIALOG_REQUEST",
			publishPayload : {
				dialogId : "DIALOG",
				dialogTitle : "DIALOG FORM WITH CONTROL ROW",
				widgetsContent : [ {
					name : "alfresco/forms/Form",
					config : {
						value : {},//Addition of this property means that Control e will now have its value published to the _valueChangeOf_ topic when the form is loaded
						widgets : [ {
							name : "alfresco/forms/ControlRow",
							config : {
								title : "Control Row 3",
								widgets : [ {
									name : "alfresco/forms/controls/TextBox",
									config : {
										fieldId : "control_e",
										name : "control_e",
										label : "Control E"
									}
								} ]
							}
						}, {
							name : "alfresco/forms/controls/TextBox",
							config : {
								fieldId : "control_f",
								name : "control_f",
								label : "Control F"
							}
						} ]
					}
				} ],
				widgetsButtons : [ {
					name : "alfresco/buttons/AlfButton",
					config : {
						label : "Close dialog",
						publishTopic : "CUSTOM_TOPIC"
					}
				} ]
			}
		}
	}, {
		name : "alfresco/buttons/AlfButton",
		config : {
			label : "Show Form Dialog",
			publishTopic : "ALF_CREATE_FORM_DIALOG_REQUEST",
			publishPayload : {
				dialogId : "FORM_DIALOG",
				dialogTitle : "FORM DIALOG WITH CONTROL ROW",
				formSubmissionTopic : "SAVE_FORM_4",
				widgets : [ {
					name : "alfresco/forms/ControlRow",
					config : {
						title : "Control Row 4",
						widgets : [ {
							name : "alfresco/forms/controls/TextBox",
							config : {
								fieldId : "control_g",
								name : "control_g",
								label : "Control G"
							}
						} ]
					}
				}, {
					name : "alfresco/forms/controls/TextBox",
					config : {
						fieldId : "control_h",
						name : "control_h",
						label : "Control H"
					}
				} ]
			}
		}
	},{
        name: "alfresco/logging/DebugLog"
    }
```

This was tested against version 1.0.62

**Comment by draperd** [comment]
Thanks for raising this issue and in particular for providing such a good model for reproducing the issue. I've raised https://issues.alfresco.com/jira/browse/AKU-938 to address this and we'll tackle it in the next sprint - expect a fix in the 1.0.65 release.

**Comment by johhnywishwash** [comment]
No problem!  Will keep an eye out for it!

**Comment by draperd** [comment]
This has now been fixed in the 1.0.65 release that is now available.
